intellij/src/main/resources/META-INF/plugin.xml

```diff
@@ -83,6 +83,8 @@
     <extensions defaultExtensionNs="com.intellij">
         <codeInsight.lineMarkerProvider language="JAVA"
                                         implementationClass="motif.intellij.provider.ScopeNavigationLineMarkerProvider"/>
+        <codeInsight.lineMarkerProvider language="kotlin"
```

Comment:
KOTLIN vs kotlin?

Reply:
Idk if it's case sensitive but I went with lower case since I've seen it being used in Kotlin source:
https://github.com/JetBrains/kotlin/blob/29b23e79f32791e456a5b4a453277f0f0b3e984d/idea/resources/META-INF/android.xml#L27